src/components/govukComponents/Input.js

```diff
@@ -6,44 +6,35 @@ import ErrorMessage from './ErrorMessage'
 class Input extends Component {
   render() {
 
-    if (this.props.hint) {
-      var hintId = this.props.id + '-hint';
+    var describedBy = ''
+    this.props.describedBy ? describedBy = this.props.describedBy : ''
 
+    if (this.props.hint) {
+      const hintId = this.props.id + '-hint'
+      this.props.describedBy ? describedBy = describedBy + ' ' + this.props.describedBy : describedBy = describedBy + ' ' + hintId;
```

Comment:
These ternaries feel a little clumsy, could this be simplified? Something like:

```
var describedBy = this.props.describedBy

...

if (this.props.hint) {
    const hintId = this.props.id + '-hint'
    describedBy += ' ' + hintId
}

...

if (this.props.errorMessage) {
    const errorId = this.props.id + '-error'
    describedBy += ' ' + errorId
}
```

Followed by a default prop?


```
Input.defaultProps = {
  describedBy: ''
}
```

In your example, if you had a hint _and_ an error message, then `describedBy + ' ' + this.props.describedBy` would run in both cases and you'd have duplicated entries in there?